modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-instance/src/test/java/com/liferay/dynamic/data/mapping/data/provider/instance/internal/WorkflowDefinitionsDataProviderTest.java

```diff
@@ -170,52 +171,41 @@ public void testThrowDDMDataProviderException() throws Exception {
 		_workflowDefinitionsDataProvider.getData(ddmDataProviderRequest);
 	}
 
-	private void _setUpLanguageUtil() {
+	private static void _setUpLanguageUtil() {
 		LanguageUtil languageUtil = new LanguageUtil();
 
 		languageUtil.setLanguage(_language);
 
-		when(
+		Mockito.when(
 			_language.get(_locale, "no-workflow")
 		).thenReturn(
 			"No Workflow"
 		);
 	}
 
-	private void _setUpLocaleUtil() {
-		mockStatic(LocaleUtil.class);
-
-		when(
-			LocaleUtil.toLanguageId(_locale)
-		).thenReturn(
-			"en_US"
-		);
-	}
-
 	private void _setUpWorkflowDefinition(
 		WorkflowDefinition workflowDefinition1, String name, String title) {
```

Comment:
@JanisZhang Could you append a separate commit to this pull to SF this parameter name `workflowDefinition1` -> `workflowDefinition`? It's strange that a common method has a parameter named `xx1`.

Reply:
@dantewang sure.